src/components/panel/_macro.njk

```diff
@@ -14,12 +14,20 @@
         {% set typeClass = ' panel--info' %}
     {% endif %}
 
+    {% if params is defined and params.type == "warn-branded" %}
+        {% set containerClass = 'census-warning' %}
+    {% endif %}
+
+    {% if params is defined and params.type == "announcement" %}
+        {% set containerClass = 'census-announcement' %}
```

Comment:
I don't think the class should reference census as it could be used across any service